tools/build-plugins/src/main/kotlin/com/datadog/build/ProjectConfig.kt

```diff
@@ -17,5 +17,5 @@ object ProjectConfig {
 
     const val GROUP_ID = "com.datadoghq"
 
-    val VERSION = Version(0, 0, 4, Version.Type.Dev)
+    val VERSION = Version(0, 4, 0, Version.Type.Release)
```

Comment:
Just by curiosity, why 0.0.4 -> 0.4.0? : )

Reply:
instead of moving with patch releases, we decided to move with minor releases to be better aligned with semver guidelines.